README.md

```diff
@@ -0,0 +1,125 @@
+<a name="readme-top"></a>
+  <h3><b>Recipe App</b></h3>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** Blog App is a web application that allows users to create, read, update, and delete blog posts. Users can sign up for an account, create new blog posts, and edit or delete their existing posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Ruby on rails</a></li>
+  </ul>
+
+  <summary>Database</summary>
+  <ul>
+    <li><a href="#">postgreSQL</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add new post]**
+- **[Add new comment]**
+- **[Add new like]**
+
```

Comment:
- [ ] Kindly update your `Key Feature` as well, this project is about Recipe and food not post comment or like 😁